server/src/main/java/org/elasticsearch/monitor/metrics/NodeMetrics.java

```diff
@@ -143,7 +143,7 @@ private void registerAsyncMetrics(MeterRegistry registry) {
 
         metrics.add(
             registry.registerLongGauge(
-                "es.node.stats.indices.translog.uncommitted_size",
+                "es.indices.translog.uncommitted_size",
```

Comment:
Nit: I think this pair would look nicer if it is
`es.indices.translog.uncommitted_operations.count` and `es.indices.translog.uncommitted_operations.size`
OR
`es.indices.translog.uncommitted.operations` and `es.indices.translog.uncommitted.size` (which will match nicely the other metric pair, `es.indices.translog.operations` and `es.indices.translog.size`)
But it's not bad as it is either.